app/webpacker/controllers/dependant_select_controller.js

```diff
@@ -8,16 +8,29 @@ export default class extends Controller {
     this.populateSelect(parseInt(this.sourceTarget.value));
   }
 
+  // private
```

Comment:
There is a proposal in that sense: https://github.com/tc39/proposal-class-fields

Reply:
Yeah there's been a few different proposals for changes around defining private methods in Javascript. Not sure when they'll be finalised though, and I guess it will only be for the latest browsers?

I know this looks a bit weird from a JS point of view and really I'm just borrowing a Ruby convention here, but to me it's a huge improvement in how the code reads. I left a verbose explanation in the commit message:
> I'm also introducing a convention here of adding a `private` comment to visually demarcate which methods are meant to be part of the public interface of the class and which aren't. The methods aren't *actually* private in any technical sense, it's just a bit of syntactic sugar to allow the intention to be communicated more clearly in the code's structure.

Reply:
Yeah, that was just for information, nothing else :)